src/layer/form-configuration/index.md

```diff
@@ -0,0 +1,205 @@
+# Attributes Form Configuration
+[[toc]]
+
+In addition to setting up [edit widgets](../form-widgets/) and designing the [form layout](../form-layout/), extra configuration can be done to the fields in QGIS to make the data collection easier and more consistent.
+
+
+<YouTube id="jc4p1PpXj3k" />
+
+
+## Default values
+Default values can be used to automatically record, e.g, the name of the surveyor, date and time of the survey, latitude and longitude of the feature, or to have frequently used values filled in advance (see <QGISHelp ver="latest" link="user_manual/working_with_vector/vector_properties.html#default-values" text="QGIS documentation" />).
+
+The default value can be a text, number or a QGIS expression. The data format of the field has to match the result of the default value expression you want to use.
+
+In QGIS, default values can be set up in the **Attributes Form** tab in **Layer Properties**. In the **Defaults** tab, you can enter the default value of the selected field (manually or using an expression). Check the option *Apply default value on update* :heavy_check_mark: to update the field value whenever there is a change in the feature.
+
+![Default values in QGIS attributes form](./qgis-form-default-values.jpg "Default values in QGIS attributes form")
+
+
+::: tip Example project available
+Various usage of default values can be explored in more detail by downloading or cloning this public project <MerginMapsProject id="documentation/form_setup" />.
+:::
+
+### Recording usernames and timestamps automatically
```

Comment:
How about not using the word "timestamp"? It sounds a bit too IT, ignore if it is a normal thing non-IT people understand.

```suggestion
### Recording usernames, dates and times automatically
```